app/models/agents/lifx_pulse_lights_agent.rb

```diff
@@ -0,0 +1,64 @@
+module Agents
+  class LifxPulseLightsAgent < Agent
+    include FormConfigurable
+    include Oauthable
+    valid_oauth_providers :'lifx'
+    
+    cannot_be_scheduled!
+    cannot_create_events!
+
+    description <<-MD
+      Pulses your your [LIFX lights](http://lifx.com/) between their current color and a color you specify.
+      
+      To be able to use this Agent you need to authenticate with LIFX in the [Services](/services) section first.
+
+      Read more about the [LIFX HTTP API](http://api.developer.lifx.com/) 
```

Comment:
You're suggesting replacing the link to http://api.developer.lifx.com/ with a the pulse-specific URL, correct? The part that I really wanted to point Huginn users at was how to construct the light selectors, but the pre-populated form control goes a long way to solving that. Do you feel strongly about it one way or the other?